dotnet/ios/InternSampleLogin/CircleView.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using UIKit;
+using CoreGraphics;
+
+namespace CodeOnlyDemo;
+
+class CircleView : UIView
+{
+	public override void Draw(CGRect rect)
+	{
+		base.Draw(rect);
+
+		//get graphics context
+		using (var g = UIGraphics.GetCurrentContext())
```

Comment:
```suggestion
		using var g = UIGraphics.GetCurrentContext ()
```
In a more current C# way of doing things, let's remove those parentheses and get rid of the curly braces (and indentation below)